### ISSUE TITLE
Make the repository more attractive and welcoming for contributions

### ISSUE THREAD

**Issue by yanna92yar** [yanna92yar/open-listings#19]
This is a no-code contribution, I'm a developer as you can guess but not only, I'm so bad at presenting things.
I believe this project deserves way more attention, and it feels sad to let this project go unseen.

It's about 7 to 10 thousands lines of code, a huge amount of effort making this happen, as said already in Readme, the project doesn't follow a canonical or a common structure of code and doing like we can see in most modern projects (MERN stack or so)... it is yet extremely performant to handle hundreds of thousands of listings and users and actions etc on an average machine.

The big stone in the shoe is the few bugs still in several parts of the app. Probably some parts of code are not well organised and presented.

To sum up, if you feel like revising the README file, drop some pictures, adjust issues, spread it to the world from your circles, please do, it gives the project probably the minimal viable seed to revive it.
